src/app/snapp_test_transaction/README.md

```diff
@@ -0,0 +1,702 @@
+            null,
+            null,
+            null,
+            null,
+            null,
+            null],
+          delegate:null,
+          verificationKey:{verificationKey:"4X1tWgjtpm8S5PqrYuTtmeUXriBHh6ZbQDauTP4nzc53M4yCnQU313c6i1hMgW15X2VyXDjZUSZuKoaoFzVEbYv37DvPPcdscsivunfSLbvwLeriqV1GXy3vbCpGNoLHnXvTFSarANqYj9vgtvnL9qJ1PmntbJUMJKXhutwRbBar31FkvnBVcti1G3ckDyKyiRXpBdykeVTLBZW5F6eoi6pM4wEax5vc4xYzDsRKYMngXwmf86hJ6ioPAjwzUyiwfUqAaLBjj2zM5eN5bQWJKp9w3uBpQDrhqcgK4NGDxfxy4KiDq1iG9gfcKvcxrJbkQ2akKP5H8a7tqUJL5FN7e5mzbbZBUnCLBcfhPNT1ZKdxbWLpkxJpSrE8QerYo7EhvR9cYbefNuMYyqXsiaFGjiNqTmCULnqhPPdDHRf6zGKN1fgREPSzTJehK39oDudAoLtyGBwxAWqurKK2CFC99SggqsfpLhfecf1s1muWwJiE3sX8Ymh6KmKi5CDdUZCpaNJytxSbHcegK9gBb679L1ckGGi9Uy4URtoDXTww8jJb2WXUJ8hN9JaNJDH3Q5j68U5CMJhnN67YihwzxFp7E6BZZkRdjPgyjFCCNxLRx7MZHaiPnpbKQ1saXo4jdN7DQdJwyVrfDxNCr8jk9mca2BSQ1UsRtq7nqSCKM2Sp4NE3CE2kJJuA57oaipqYuTwxAVmoXoncZKGmbgUVRnUWygecV949hETQNn6qQyUWsapLKxTVKXAvsRpsmWEatd1iRqnG1t4Zu9cWgoy45LGBFRXtKQVfNy1Q7fGQCkVnsEHmLWZfSSMDVZDDoyfpcUfZBfRaaZrK3E5RosrPeZKvUqhwKJU4uzJUCuggVDspDqLEQhj9WZGrPv7gDev7gd7Sn8YPDtcZDtbeTwtar8ZyPBd6acssjQK8cKZiKmU3NzDNcaZYimHah1GDsXVpdEJLNDqXCsp9RvM27QpsgeX1c9rzyDdpPFDX6d1a3xtLHC6GGE4mLWMn2BAC52L8Gkaa7BUKKh2NFnnBK9tVxYQmX7qS8mh6uteqKBSA8VyLMbaX9t21SM5PC7uVZHKByQz42MMiJ4sUPYRzXn1Piabqo42RcJ2vvT6K2Dzz1M9FLPZTCgPUERR5wCoDZYJdHb8EN7TET9L16LpLhVGsSkJ3FcA2CtwzS8UCRgBPVXDmy2WBzZYC4oarJCCJDmRmimE6aUmrUCP5HPdXxAcGo5t2Yc5qukZLAc7YRXk77ATUYZAXvwFcdYRr7tsAsmT4P9dGZkVSHYwR6jX7kLVcHE6UNpd7WUTQF3zdz6rQVNzpTVsHHRAjVRHTTbMyq43cUf3rs8CzQ2Kxk9TzE22LCA8UyKYtwZmVp5hwRPnPLuRWUaUKp8DfJ3J6KvUJxrF3LM747i9uFUY2g3tnLLhsfXrLaZwBd5Lvap4xE61daiDiaUtVwucunQD9EeNtY5v1w9y7y1umCj1tKcT8fdkbcj8heJPsHijjjTQvKUM5s1KG2KgKQTmGj2bx4asHrsgKHuctpzj49ozXYwAdh5iTAYY3MqwTE6whLPFpdXzH4DgYCLAkCB2yn7QZdYJNtsoN2pJgsQgwbGZ32G3KvGS6Ufvmtgj2txJ9yhwA9r9oLaKjN1yAXP3Lp7ydyqgRLHcByokeXj3ErAkgZ2wb3yYswgVWXE1CbuTTtdKZtUKSPMZTfKuVxEtwfz3Cm2g3dmCCnJgB3ogAacLMSJifRokDLtoKsByECsvxQGH4qoZrxDBDNqV8disJnznULXb5gcaVeVBvs97MYWbaYuESkmrdjfPNhKrUVToqyFCZNoKWY2ruKhjkyCCPjwY8mxSZ3Bqu1DsGbrCrDmpjYVKDYhRxxGRtdcDRFGuLiLZ7bqQep9Dm5NsVY4r3jaJARWNXhshpLLkrDa2VNnmADTgPyFWUtysTwX23m1wRx7Wx1CHJVn6KK9rFGumTsSUA2eZ3F8Fa9dxRiNoufepyMk8mYVv9woNEDow4f1pQv5k6h26dy2R8h7Bec7DGDVpkgUiSZHYe9qDdUhoVCCMqsbaB6pqfRpQ2MCXkvoshz5VHX1tawQexpfscFoyC8amTE61veMyRgxY35ZgWypVK9QQSpnRFQg8YDJLxr7FFwzCxZqJbRZRd2w3AaRLkeHjD6zZbRXHtd7qPJyR2MfAEtzACYDKDcNAzgdN5zN2XcuCVQjqkdHv2E7dR2J4hJYjcorTh4p2s5aBymPunN6VcaKPUQJyQn7xv2TYqF8tZ1GDeQwF9jkKzWFqgGNuDVGgmLPAivEYvLSWVPyDAnNbFwRBegXXSRoUyM73MQS1wKwsou6emBgPDXQySAV4ak91S2vvvEmnEy8BCBz6cnFCiiaLoiBTxkpw4gMEowiZxmA8zMpwaMUaSzqLSc8tKXjjQ9Q9gZi79zRQ81QowxxPS4rn2MEBRUuvvwp9Hug4z19Z3pz7gC9RBjumEaniNjDLbdc8", hash:"0x18F78C4C0BE6CC4E27B232CF5B41ADAA63D881ABAA68155AA5035769C2A11C85"},
+          permissions:{stake:true,
+            editState:Proof,
+            send:Signature,
+            receive:None,
+            setDelegate:Signature,
+            setPermissions:Signature,
+            setVerificationKey:Signature,
```

Comment:
I'm thinking about changing all of these to `Proof` for the update-all-fields test, would that be OK?

Reply:
Or maybe I should have a separate step to update the permissions

Reply:
Would prefer making a separate transaction to update the permissions